addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -3857,12 +3857,14 @@ def test_post_valid_invoices_when_auto_post(self):
         self.assertEqual(invalid_invoice_1.state, 'draft')
 
         self.assertTrue(any(
-            message.body == "<p>The move could not be posted for the following reason: The field 'Customer' is required, please complete it to validate the Customer Invoice.</p>"
+            message.body == "<p>The move could not be posted for the following reason: The 'Customer' field is required to validate the invoice.\n"
```

Comment:
Replace "<p>The move could not be posted for the following reason: The 'Customer' field is required to validate the invoice.\n" by:
"Please provide a customer to validate the invoice.\n" (keep the second part)